.idea/uiDesigner.xml

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
```

Comment:
@maria-salmasian there is no need to push this files, put them on gitignore

Reply:
adde to git ognore